src/index.js

```diff
@@ -25,7 +25,10 @@ client.on('message', async message => {
     message.react('😇');
   }
 
-  if (channel.name && channel.name === database.getConfig('channel') && message.cleanContent.startsWith(database.getConfig('prefix'))) {
+  // Check that we are not in private messages
+  if (!channel) return;
```

Comment:
`channel` could be one of `TextChannel, DMChannel, GroupDMChannel`. It should not be falsy.

I don't think this will do what the comment says it will. Instead, check that `channel.guild` is not falsy. What would be even better is to check `channel.type === 'text'` to make sure it's a guild text channel.

Reply:
After some testing, the bot will still work as intended since you can't add the bot to a group DM, which is the only other type that `channel` can be that also has a name. However, the comment is still misleading and the code isn't following good practices.